lib/vagrant-openshift/action/install_openshift_base_dependencies.rb

```diff
@@ -104,11 +104,18 @@ def call(env)
 
 if [[ ${GO_VERSION[2]} == "go1.4"* ]]; then
   GOPATH=/data go get golang.org/x/tools/cmd/cover
-else
-  GOPATH=/data go get code.google.com/p/go.tools/cmd/cover
-fi
 
-GOPATH=/data go get golang.org/x/tools/cmd/vet
+  GOPATH=/data go get golang.org/x/tools/cmd/vet
+
+  # TODO: Remove for go1.5
```

Comment:
This comment (poorly written) meant that `go vet` is moving into `go1.5` and none of this will be necessary on the upgrade. This won't apply to `go cover`.